File: src/main/java/com/rs/java/game/area/multi/BarbarianVillageMulti.java
package com.rs.java.game.area.multi;

import com.rs.java.game.WorldTile;
import com.rs.java.game.area.Area;
import com.rs.java.game.area.shapes.Rectangle;

public class BarbarianVillageMulti extends Area {

	public BarbarianVillageMulti() {
		super(new Integer[]{12341},
				// North of falador party room
				new Rectangle(new WorldTile(3048, 3392, 0), new WorldTile(3055, 3407, 0)),
				//west of barbarian village
				new Rectangle(new WorldTile(3056, 3408, 0), new WorldTile(3071, 3447, 0))
		);
	}

	@Override
	public Area update() {
		return this;
	}

	@Override
	public String name() {
		return "Barbarian Village Multi";
	}

	@Override
	public boolean member() {
		return false;
	}

	@Override
	public Environment environment() {
		return Environment.MULTI;
	}
}

File: src/main/java/com/rs/kotlin/game/world/area/areas/multi/BarbarianVillageMulti.kt
package com.rs.kotlin.game.world.area.areas.multi

import com.rs.java.game.WorldTile
import com.rs.kotlin.game.world.area.Area
import com.rs.kotlin.game.world.area.shape.Rectangle

class BarbarianVillageMulti : Area(
    arrayOf(12341),  // North of falador party room
    Rectangle(WorldTile(3048, 3392, 0), WorldTile(3055, 3407, 0)),  //west of barbarian village
    Rectangle(WorldTile(3056, 3408, 0), WorldTile(3071, 3447, 0))
) {
    override fun update(): Area {
        return this
    }

    override fun name(): String {
        return "Barbarian Village Multi"
    }

    override fun member(): Boolean {
        return false
    }

    override fun environment(): Environment {
        return Environment.MULTI
    }
}